FAQ.md

```diff
@@ -315,12 +315,14 @@ cases they may be used as `dockerBuild` hosts too.
 
 Instructions on how to create a static docker container can be found [here](https://github.com/adoptium/infrastructure/tree/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/README.md)
 
+### Dockerhost Patching
+
+At the moment we have the [updatepackages.sh](https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/scripts/updatepackages.sh) script which runs weekly on all of our Dockerhost systems to ensure that the Static Docker containers are patched and up to date. The script is also used to install new test [prerequisite](https://github.com/adoptium/aqa-tests/blob/master/doc/Prerequisites.md#prerequisites) packages onto the containers.
```

Comment:
Can you update this to make it clear how the regular update is done (i.e. via AWX rather than a cron job)